src/elvis_style.erl

```diff
@@ -833,6 +853,21 @@ is_ignored_var(Zipper) ->
         _OtherType -> false
     end.
 
+-spec check_exec_path_length(elvis_code:tree_node(), integer()) ->
+    [elvis_result:item_result()].
+check_exec_path_length(Node, MaxLength) ->
```

Comment:
For this work, https://github.com/inaka/elvis/blob/a79dc5f505d0678cddb06fd8b37d5f160c7a0009/src/elvis_code.erl#L185-L274 must be migrated to `ktn_code`